notarymysql/migrate.sql

```diff
@@ -0,0 +1,18 @@
+-- This migrates initial.sql to tables that are needed for GORM
+
+ALTER TABLE `tuf_files`
+ADD COLUMN `created_at` timestamp NULL AFTER `id`,
```

Comment:
We had some bugs with just setting `NULL`, it turned out, as redundant as it seems, these need to be `NULL DEFAULT NULL` for MySQL to behave as expected. It's something special to do with the timestamp type, can't remember exactly what is was now.